tutorials/23_MultiHop_QA_With_Agents.ipynb

```diff
@@ -0,0 +1,5680 @@
+            "695868cd0d644e0badf11f793af53c2d",
+            "fa0bf4ca5f9b406082a7d1a6dccb32b5",
+            "e22644ca58294d4dbcddd208583886b9",
+            "272225718b18418f9d9832662abdca00",
+            "3d85d0886e1f4e2fb9b812e89fa34bd8",
+            "c594db0163e64685929954ce4b4c0a6e",
+            "896b831869b24f3bb06a611375e0feea",
+            "110e84e5a4344d27b750e272e24ca523",
+            "d6e19afe43c640e894ca47f9a38fbc60"
+          ]
+        },
+        "id": "n9bmrIUCatMk",
+        "outputId": "9f110081-72ce-4f80-fea0-10b3f8a61654"
+      },
+      "outputs": [],
+      "source": [
+        "from haystack.nodes import BM25Retriever, FARMReader\n",
+        "from haystack.pipelines import ExtractiveQAPipeline\n",
+        "\n",
+        "retriever = BM25Retriever(document_store=document_store)\n",
+        "reader = FARMReader(model_name_or_path=\"deepset/roberta-base-squad2\", use_gpu=True)\n",
+        "presidents_qa = ExtractiveQAPipeline(reader=reader, retriever=retriever)\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "mgQAP-4T7Fh6"
+      },
+      "source": [
+        "### 3) Ask a question!\n",
+        "\n",
+        "Now that you have an ExtractiveQAPipeline, go ahead and ask it a question about the presidents of the USA. \n",
+        "\n",
+        "Below is the question: \"Who was the 44th president of the USA?\"\n",
+        "\n",
+        "To start seeing the limitations of an ExtractiveQAPipeline on it's own, you may want to try a question like: \"Who is the wife of the 44th president of the USA?\".\n",
+        "\n",
+        "Notice hoe this is 2 questions in one. An extractive model will strugle to find the answer to this question unless the answer is phrased clearly in our documents. For example: \"The wife of the 44th president is Michelle Obama\"."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 930,
+          "referenced_widgets": [
+            "d31a483cf6e443a5891ca14672ffed3b",
+            "a0dcb1d860704bbdb54447dfd0cc92bd",
+            "5e85990cb7374ddc9f7107a72f812e5e",
+            "acf1cdc5c6484d8ab5f78ac217656b17",
+            "083cb8b3aa1240129a75e5c2a7eacbb6",
+            "7f4d17a4998f44b58f813b815d16f7f8",
+            "6ffba94cc23846209c4629cb10bce042",
+            "de52fc90bd3f4db0b540535246b33c4f",
+            "acd5844ab2d64071a1d59a1df7baafbd",
+            "744746530a4e401bb6e9e28d89e54eed",
+            "3bb3e922fc8c471a8c1aaaf5bbca2d92"
+          ]
+        },
+        "id": "HdS-satXbSsw",
+        "outputId": "751e30c8-50ae-48d3-8e34-df7023d1252a"
+      },
+      "outputs": [],
+      "source": [
+        "from haystack.utils import print_answers\n",
+        "\n",
+        "result = presidents_qa.run(\"Who was the 44th president of the USA\")\n",
```

Comment:
I'd show the failing query here to highlight the need for multihop/agents and have the other commented out